Module4_Labs/Lab3_File_System/Cards/21.md

```diff
@@ -10,4 +10,29 @@ The first element of a tuple represents the "from" end of an arrow, while the se
 
 Now would be a great time to contemplate how the "from-to" relationship is represented in the input file and use that information to contemplate the direction of your arrows.
 
-![Please only contemplate the directions of your edges, not your life. Existential crises are not part of this lab.](https://images.pexels.com/photos/38640/directory-traffic-note-shield-38640.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260)
\ No newline at end of file
+![Please only contemplate the directions of your edges, not your life. Existential crises are not part of this lab.](https://images.pexels.com/photos/38640/directory-traffic-note-shield-38640.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=750&w=1260)
```

Comment:
Sorry I probably should've been more clear in the issue. There should be no code in medium cards, only hard cards, so simply give the student some more general hints not direct code.

Reply:
I resolved the issue. I actually put the code back into 211.md since it's the easy card and just gives away the answer. If I were to put the code in the hard card, it would kind of mess up the flow of the ordering of difficulty of each sub-card for Card 2.